t/plugin/syslog.t

```diff
@@ -132,8 +132,36 @@ passed
 [error]
 
 
+=== TEST 5: add plugin metadata
```

Comment:
Could you check the logged data format?
You can refer to https://github.com/apache/apisix/pull/7328/files#diff-1108dce1e7823ca0b746c4ba69e692e4b02c6d49117d70859954b56cbb810c29 as an example.

BTW, new test should be added at the end.

Reply:
I fixed test reordering problem.

Should I add "access request" test?
And is that enough if is require?
```
=== TEST 10: access for log format
--- request
GET /hello
--- response_body
hello world
```

Reply:
Let's do it. And we need to inject some code in `extra_init_by_lua` to check the logged data.

Reply:
I think we don't need `extra_init_by_lua` for these tests.
I added new test cases. Please review it again.

BTW: It was to hard to understand test framework for me. :)

Reply:
> BTW: It was to hard to understand test framework for me. :)

You can read the doc in https://github.com/apache/apisix/blob/master/docs/en/latest/internal/testing-framework.md